post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -22,6 +22,6 @@ include::modules/multi-architecture-modify-machine-set.adoc[leveloffset=+2]
 .Additional resources
 * xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc[Creating a compute machine set on Azure] 
 
-include::modules/multi-architecture-upgrade-mirrors.adoc[leveloffset=+1]
```

Comment:
should we remove the multi-architecture-upgrade-mirrors file?

Reply:
I believe since the modules/multi-architecture-upgrade-mirrors.adoc file still exists in main, for the 4.12 docs, Github just thinks that I renamed the file bc its in the same place. The upgrade section does not appear in the render

Reply:
@aleskandro I spoke with some folks and decided to remove the file 👍 . I just updated that in this PR